app/src/main/AndroidManifest.xml

```diff
@@ -40,6 +40,10 @@
                 <data android:mimeType="text/plain" />
             </intent-filter>
         </activity>
+        <meta-data android:name="io.sentry.dsn" android:value="https://8962f581b7aa451d941583f4bb1f745c@o1124524.ingest.sentry.io/6162821" />
+        <!-- Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
+           We recommend adjusting this value in production. -->
```

Comment:
Clearly we don't want to merge this as is

Reply:
The `tracesamplerate` could/shouldmust be set to 0.

Sentry.io could give us java exception when the application crash during "real" test;
The data are reported to the sentry.io platform.

We certainty could use it only in debug mode or private release and remove it from google and f-droid release.